library/Zend/Filter/Word/Service/SeparatorToSeparatorFactory.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Zend\Filter\Word\Service;
+
+use Zend\ServiceManager\Exception\ServiceNotCreatedException;
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use Zend\Filter\Word\SeparatorToSeparator;
+
+class SeparatorToSeparatorFactory implements FactoryInterface
+{
+    /**
+     * @var array
+     */
+    protected $options;
+
+    /**
+     * Constructor.
+     *
+     * @param array $options
+     */
+    public function __construct(array $options = array())
```

Comment:
Hint: factory must implement the [`MutableCreationOptionsInterface`](https://github.com/zendframework/zf2/blob/9d86b442bddd728612280bddcfe116d368367cea/library/Zend/ServiceManager/MutableCreationOptionsInterface.php)